test/test_views.py

```diff
@@ -31,11 +31,12 @@ def _create_storage(self):
     def _create_blogging_engine(self):
         return BloggingEngine(self.app, self.storage)
 
-    def setUp(self):
+    def setUp(self, **config):
```

Comment:
good idea.  i will change

On Sat, Jul 23, 2016 at 1:35 PM, Goutham Balaraman <notifications@github.com
> wrote:

> In test/test_views.py
> <https://github.com/gouthambs/Flask-Blogging/pull/73#discussion_r71977892>
> :
>
> > @@ -31,11 +31,12 @@ def _create_storage(self):
> >      def _create_blogging_engine(self):
> >          return BloggingEngine(self.app, self.storage)
> >
> > -    def setUp(self):
> > +    def setUp(self, **config):
>
> The UnitTest standard signature for setUp seems to be be just setUp(self)
> per unittest docs
> <https://docs.python.org/2.7/library/unittest.html#unittest.TestCase.setUp>.
> One workaround is to implement other_config() method which will just
> return an empty dict in the parent class but can be overridden in
> TestViewsWithBootstrap to inject the required parameters.
>
> More than anything, I am concerned if this could have the wrong setUp to
> be called down the line.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/gouthambs/Flask-Blogging/pull/73/files/6bc186a6713454d4b0b05c486aab39970533d51c..e8257e28afc601d672b89a428cbd547a4e95de4a#r71977892>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABy59bmReS-xd-K25HgEgVUeYd24prDIks5qYnsXgaJpZM4JIOM->
> .
>